app/src/components/ChoiceCard.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import ChoiceCard from './ChoiceCard'
+
+import '@testing-library/jest-dom';
+
+test('should render: title. Should not render: paragraph, image', () => {
```

Comment:
I think alias **it** would fit well here (instead of test), so the reading would be like "it should ...". 
No need for change, just a thought.

Reply:
```suggestion
test('It should render: title. It should not render: paragraph, image', () => {
```